dallinger/recruiters.py

```diff
@@ -779,7 +995,13 @@ def __init__(self, *args, **kwargs):
         self.notifies_admin = admin_notifier(self.config)
         self.mailer = get_mailer(self.config)
         self.store = kwargs.get("store") or RedisStore()
-        self._validate_config()
+
+        not_validate_config = (
+            "not_validate_config" in kwargs and kwargs["not_validate_config"]
+        )
+
+        if not not_validate_config:
```

Comment:
Maybe better just to put `not_validate_config` in the `__init__` signature? Or better, put `validate_config=True` in the `__init__` signature?

Reply:
Yes, `not_validate_config` is now a keyword argument.

Reply:
OK, if you prefer it this way, then maybe it is more idiomatic if you call it `skip_config_validation`?

Reply:
Yep, did it see below.